aws_lambda.py

```diff
@@ -143,13 +129,45 @@ def timestamp_to_date(timestamp_milliseconds):
 
 def refresh_signature(event, context):
     server_url = event['server']
-    collections = event.get('collections', DEFAULT_COLLECTIONS)
     auth = tuple(os.getenv("REFRESH_SIGNATURE_AUTH").split(':', 1))
 
-    for collection in collections:
+    # Look at the collections in the changes endpoint.
+    bucket = event.get('bucket', "monitor")
+    collection = event.get('collection', "changes")
+    client = Client(server_url=server_url,
+                    bucket=bucket,
+                    collection=collection)
+    print('Looking at %s: ' % client.get_endpoint('collection'))
+    changes = client.get_records()
+
+    # Look at the signer configuration on the server.
+    info = client.server_info()
+    signed_resources = info['capabilities']['signer']['resources']
+
+    def get_source(change):
+        # Small helper to pick the source collection from the collection
+        # mentionned in the changes endpoint. (eg. blocklists/plugins -> staging/plugins)
```

Comment:
`mentioned`. Also, I would probably say `get` instead of `pick`, or maybe `find` or `identify`.